### ISSUE TITLE
Set proxy credentials from functional tests

### ISSUE THREAD

**Issue by igor-sirotin** [status-im/status-go#6144]
1. Read these env vars:
    - `STATUS_BUILD_PROXY_USER `
    - `STATUS_BUILD_PROXY_PASSWORD `

2. Pass values to these endpoints as part of `WalletSecretsConfig`:
    - `CreateAccountAndLogin`
    - `RestoreAccountAndLogin`
    - `LoginAccount` 
    
    https://github.com/status-im/status-go/blob/29bf0d6c8e34bc3663a3ed9678061a8b215238db/params/config.go#L551-L556

    Reference status-desktop for correspondence of env vars to parameters:
    https://github.com/status-im/status-desktop/blob/73af1b8ba0fd82a5259a5799fef3bd03280869b3/src/app_service/service/accounts/service.nim#L168-L172

    ```nim
      proc buildWalletSecrets(): WalletSecretsConfig =
        return WalletSecretsConfig(
          # ...
          statusProxyMarketUser: STATUS_PROXY_USER_RESOLVED,
          statusProxyMarketPassword: STATUS_PROXY_PASSWORD_RESOLVED,
          statusProxyBlockchainUser: STATUS_PROXY_USER_RESOLVED,
          statusProxyBlockchainPassword: STATUS_PROXY_PASSWORD_RESOLVED,
        )
    ```

3.  Also set `WalletSecretsConfig.statusProxyStageName` to `"test"`.

**Comment by igor-sirotin** [comment]
@antdanchenko raised a risk of potential leak of credentials in functional test logs, if we pass them through API.

I discussed it today with @osmaczko. 
Even if we embed it to status-go during build time, anyway there's always a risk of leaking it in `geth.log` itself. 

So we agreed not to overcomplicate things now and **pass the credentials through API**. 
At the same time, we must implement right tools to prevent leaks. We discussed these options:
- https://github.com/status-im/status-go/issues/5678
- On tests teardown, read `geth.log` and check if it contains any known credentials
